share/vss/vss.go

```diff
@@ -0,0 +1,579 @@
+// vss.go implements the verifiable secret sharing scheme by Pedersen recalled
+// in "Provably Secure Distributed Schnorr Signatures and a (t, n) Threshold
+// Scheme for Implicit Certificates"  by Stinson and Strobl.
+// The basic scheme is taken from Pedersen's paper: "Non-interactive and
+// information-theoretic secure verifiable secret sharing", in Crypto ’91, pages
+// 129–140, 1991.
+// At a higher level, you have one party called the Dealer who wishes to share a
+// secret securly between other parties, called the Verifiers. In this scheme,
+// neither the verifiers or the dealer is deemed trustworthy, but rather, the
+// assumptions is that the adversary has at most t verifiers amongst n, or the
+// dealer is malicious. In that case, there needs to be some way for Verifiers to
+// check the correctness of the shares they receive. This is the scheme goal.
+// 1) The dealer send a Deal to every verifiers using `Deals()`
```

Comment:
Yep I just removed 
> and successfully finish the protocol run.
Because they don't have to reveal it now as soon as the shares are certified ;)